app/services/UIBuilder.scala

```diff
@@ -241,38 +241,21 @@ class UIBuilder {
   private def fromSectionAndNoteCallout(caption: Text, nc: NoteCallout)(implicit ctx: UIContext): UIComponent =
     Details(caption, Seq(TextBuilder.fromPhrase(nc.text)))
 
-  private def fromNonExclusiveSequence(nonExclusiveSequence: NonExclusiveSequence, components: Seq[UIComponent])(implicit ctx: UIContext): UIComponent = {
-    val (errMsgs, uiElements) = partitionComponents(components, Seq.empty, Seq.empty)
-
-    val (text, hint) = TextBuilder.fromPhraseWithOptionalHint(nonExclusiveSequence.text)
-    val options: Seq[Text] = nonExclusiveSequence.options.map{phrase => TextBuilder.fromPhrase(phrase)}
-
-    ui.NonExclusiveSequence(text, hint, options, uiElements, errMsgs)
-  }
-
-  private def fromExclusiveSequence(exclusiveSequence: ExclusiveSequence, components: Seq[UIComponent])(implicit ctx: UIContext): UIComponent = {
-
-    val (errMsgs, uiElements) = partitionComponents(components, Seq.empty, Seq.empty)
-    val (text, hint) = TextBuilder.fromPhraseWithOptionalHint(exclusiveSequence.text)
-    val options: Seq[Text] = exclusiveSequence.nonExclusiveOptions.map{phrase => TextBuilder.fromPhrase(phrase)}
+  private def fromSequence(sequence: Sequence, components: Seq[UIComponent])(implicit ctx: UIContext): UIComponent = {
+    val answers = sequence.options.map { ans =>
```

Comment:
There is validation in the backend which prevents guidance containing multiple execlusive answers (generates a MultipleExclusiveOptions back to the Ocelot user if they try to push/preview a process). So code needs only to find and process the single exclusive

Reply:
It depends if the UI sequence model is changed then that this will be changed with it to a partition (using the first of the exclusive options)

Reply:
dont understand your answer???